### ISSUE TITLE
0.55.0 pre-release thread (stuff to do to be shipped in the release)

### ISSUE THREAD

**Issue by illwieckz** [Unvanquished/Unvanquished#3087]
In the following, the “future” branch means the `for-0.55.0/sync` one, as compared to the `master` one.

- Whole
  * [x] Produce and test release candidates with a full rebuild of the data, game and engine from future branch
  * [x] Host a server with the release candidates
- Engine
  * TODO:
  * [x] https://github.com/DaemonEngine/Daemon/pull/1293
  * [x] Straighten out overbright cvars https://github.com/DaemonEngine/Daemon/issues/1289
  * [x] Fix https://github.com/DaemonEngine/Daemon/issues/1297
  * [x] Merge the future branch to master
- Game
  * TODO:
  * [x] Merge the future branch to master
- Translation
  * [x] Temporary configure Weblate to track the future branch, to translate what's only in the future branch  
  * [x] Undo it once the future branch is merged to master
- Assets
  * TODO:
  * [x] https://github.com/Unvanquished/Unvanquished/issues/2880
  * [x] Merge the human IQM (may require some game code to be merged)
  * [x] Merge the new creep texture
  * [x] Audit particle systems to fix ones that look bad following the [autosprite rework](https://github.com/DaemonEngine/Daemon/pull/1281) and [heat haze fix](https://github.com/DaemonEngine/Daemon/pull/1342).
  * [x] Merge the low models in future branch
  * [x] Merge the future branch to master

**Comment by slipher** [comment]
My wish list for an RC, from currently open PRs that I believe are usable in their current state:
- https://github.com/UnvanquishedAssets/res-weapons_src.dpkdir/pull/22
- https://github.com/UnvanquishedAssets/res-weapons_src.dpkdir/pull/24
- https://github.com/UnvanquishedAssets/res-weapons_src.dpkdir/pull/25
- https://github.com/DaemonEngine/Daemon/pull/1350
- https://github.com/DaemonEngine/Daemon/pull/1351 (Important, this one is a showstopper!)
- https://github.com/Unvanquished/Unvanquished/pull/2977
- https://github.com/Unvanquished/Unvanquished/pull/2975 + https://github.com/UnvanquishedAssets/res-players_src.dpkdir/pull/19
- https://github.com/Unvanquished/Unvanquished/pull/2894

**Comment by DolceTriade** [comment]
My wishlist:
 - [ ] Psaw -- https://github.com/UnvanquishedAssets/res-weapons_src.dpkdir/pull/26
 - [ ] Chaingun
 - [ ] Freshly exported IQM human
 - [ ] Creep texture -- https://github.com/UnvanquishedAssets/res-buildables_src.dpkdir/pull/22

**Comment by slipher** [comment]
https://github.com/Unvanquished/Unvanquished/pull/3105 needs a review. It's very simple.

https://github.com/DaemonEngine/Daemon/pull/1322 might be nice to have. It doesn't change anything unless you use the cvar.
